packages/components/CHANGELOG.md

```diff
@@ -18,9 +13,9 @@
 -   `Composite`: make items tabbable if active element gets removed ([#65720](https://github.com/WordPress/gutenberg/pull/65720)).
 -   `DatePicker`: Use compact button size. ([#65653](https://github.com/WordPress/gutenberg/pull/65653)).
 -   `DropZone`: fix class names on drop ([#65798](https://github.com/WordPress/gutenberg/pull/65798)).
-
-### Enhancements
-
+-   `Tooltip`: add `aria-describedby` to the anchor only if not redundant ([#65989](https://github.com/WordPress/gutenberg/pull/65989)).
+-   `PaletteEdit`: dedupe palette element slugs ([#65772](https://github.com/WordPress/gutenberg/pull/65772)).
```

Comment:
Were these two fixes cherry-picked into the branch, too? The rest looks great.

Reply:
These are moved from the above. They seem to be backported too!

What should we do if these were not backported, though? Should we leave them in the "Unreleased" section or remove them?

Reply:
I would remove them completely if not cherry-picked before.